Corgibytes.Freshli.Cli/Functionality/ReadCycloneDxFileFromFileSystem.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.IO;
+using Corgibytes.Freshli.Cli.Resources;
+
+namespace Corgibytes.Freshli.Cli.Functionality;
+
+public class ReadCycloneDxFileFromFileSystem : IReadFile
+{
+    public JsonCycloneDx ToJson(string filePath)
+    {
+        try
+        {
+            var stream = new StreamReader(filePath);
```

Comment:
I think that this needs to be changed to a ["using declaration"](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement) so that the file is properly closed and disposed of when the object gets garbaged collected. 

```suggestion
            using var stream = new StreamReader(filePath);
```